docs/features/ValueTuples.cs

```diff
@@ -24,8 +24,89 @@ public interface ITuple
         object this[int i] { get; }
     }
 
-    public static class ValueTuple
+    public struct ValueTuple : IEquatable<ValueTuple>, IStructuralEquatable, IStructuralComparable, IComparable, ITupleInternal, ITuple
     {
+        public override Boolean Equals(Object obj)
+        {
+            return ((IStructuralEquatable)this).Equals(obj, EqualityComparer<Object>.Default);
+        }
+
+        public Boolean Equals(ValueTuple other)
+        {
+            return true;
+        }
+
+        Boolean IStructuralEquatable.Equals(Object other, IEqualityComparer comparer)
+        {
+            if (other == null || !(other is ValueTuple)) return false;
+
```

Comment:
Knowing that ValueTuple is a struct, I wonder if    ```other == null || ``` is needed.
```is```  implies null check and we know that nothing inherits from ValueTuple